src/__internal__/input-behaviour/useInputBehaviour.spec.js

```diff
@@ -1,79 +1,109 @@
 import React from "react";
+import styled from "styled-components";
 import { act } from "react-dom/test-utils";
 import { mount } from "enzyme";
 
 import useInputBehaviour from "./useInputBehaviour";
 
-const HookTestComponent = () => {
-  const { inputRef, ...hookValues } = useInputBehaviour();
+const Input = styled.input``;
+
+// eslint-disable-next-line
```

Comment:
what do we need to disable here?

Reply:
You get a prop defintion warning without it

Reply:
oh this is spec file, ok